app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">FHIR Bundle</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+          <div class="content">
+            <pre style="white-space:pre-wrap;">
+              <p> <%= JSON.pretty_generate(@health_card.to_hash) %> </p>
+            </pre>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Minified FHIR Bundle</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <div class="content">
+           <pre>
+              <p> <%= @health_card.bundle.to_json %> </p>
+           </pre>
+          </div>
+        </div>
+      </div>
+    <br>
+    <br>
+
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">Encoded JWS</p>
+        <a class="card-header-icon card-toggle">
+          <i class="fa fa-angle-down"></i>
+        </a>
+      </header>
+        <div class="card-content">
+          <pre style="white-space:pre-wrap;">
+            <div class="content", style="word-wrap: break-word; width: 800px">
+                <p> <%= @jws_encoded_details%> </p>
+            </div>
+          </pre>
+        </div>
+      </div>
+    <br>
+    <br>
+    
+    <div class="card is-fullwidth">
+      <header class="card-header">
+        <p class="card-header-title">QR Details</p>
```

Comment:
I think this should be called something like "Numeric QR", since it's not actually showing much detail.